book-crawler/tests/test_async_api.py

```
@@ -0,0 +1,77 @@
+            print(f"   - 신규: {result.get('new', 0)}권")
+            print(f"   - 업데이트: {result.get('updated', 0)}권")
+            print(f"   - 에러: {result.get('errors', 0)}건")
+            break
+            
+        elif status == "failed":
+            print("\n❌ 작업 실패!")
+            print(f"   에러: {status_data.get('error', '알 수 없는 에러')}")
+            break
+            
+        elif status == "running":
+            current = status_data.get("current_book", 0)
```

Ruff (0.12.2)

Local variable `current` is assigned to but never used

Remove assignment to unused variable `current`

(F841)